Bio/NaiveBayes.py

```diff
@@ -62,10 +62,9 @@ def __init__(self):
 def calculate(nb, observation, scale=False):
     """Calculate the logarithmic conditional probability for each class.
 
-    - nb          - A NaiveBayes classifier that has been trained.
-    - observation - A list representing the observed data.
-    - scale       - Boolean to indicate whether the probability should be
-    scaled by ``P(observation)``.  By default, no scaling is done.
+        - nb          - A NaiveBayes classifier that has been trained.
+        - observation - A list representing the observed data.
+        - scale       - Boolean to indicate whether the probability should be scaled by ``P(observation)``.  By default, no scaling is done.
```

Comment:
It would be nice if even the summary line fits, but as PEP8 makes clear this is a guideline only. Given the indentation and the triple-quotes count to this limit, I'm comfortable with being more relaxed on the length of the summary line.